### ISSUE TITLE
Explore multiple regularization techniques to reduce overfitting and optimize them

### ISSUE THREAD

**Issue by uberVelocity** [TheVeggydude/HebrewHandwritingRecognition#10]
- batch normalization
- dropout
